mf.py

```diff
@@ -1,9 +1,15 @@
+import pycuda.autoinit
+import pycuda.gpuarray as gpuarray
+import skcuda.linalg as linalg

```

Comment:
It would be good if you can add a `requirements.txt` file to specify the dependencies. I didn't have one because it was only dependent on numpy before.

Reply:
Ok I will write a .txt file like this and upload it.